frontend/packages/ux-editor/src/utils/optionsUtils.ts

```diff
@@ -35,6 +35,26 @@ export function getSelectedOptionsType(
     return SelectedOptionsType.Unknown;
   }
 
+  if (!!options) {
+    return SelectedOptionsType.CodeList;
+  }
+
+  return componentUsesDynamicCodeList(codeListId, optionListIds)
+    ? SelectedOptionsType.ReferenceId
+    : SelectedOptionsType.CodeList;
+}
+
+// Copy of function above. Todo: Remove once featureFlag "optionListEditor" is removed.
+export function getSelectedOptionsTypeV1(
+  codeListId: string | undefined,
+  options: IOption[] | undefined,
+  optionListIds: string[] = [],
+): SelectedOptionsType {
+  /** It is not permitted for a component to have both options and optionsId set on the same component. */
+  if (options?.length && codeListId) {
+    return SelectedOptionsType.Unknown;
+  }
```

Comment:
Can `getSelectedOptionsTypeV1` and `getSelectedOptionsType` be moved next to where they are used? If we can place `getSelectedOptionsTypeV1` inside one of the "old" folders, Then we don't need to have different versions next to each other, and the old function will be deleted together with the old functionality.